src/tests/Dashboard/components/DictionariesPaginations.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import expect from 'expect';
+
+import Paginations from '../../../components/dashboard/components/DictionariesPaginations';
+
+describe('Test suite for dictionaries pagination component', () => {
+  it('Should render without breaking', () => {
+    const props = {
+      dictionaries: 10,
+      firstDictionaryIndex: 1,
+      totalDictionaries: 20,
+      currentPage: 1,
+      handleClick: jest.fn(),
+      lastPage: 2,
+      onChangeLimit: jest.fn(),
+    };
+    const wrapper = mount(<Paginations {...props} />);
+    const selectLimit = wrapper.find('select');
+    selectLimit.simulate('change', { target: { value: 25 } });
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Considering the limit change event is not the only event in the pagination component, expecting wrapper toMatchSnapshot doesn't seem like the most efficient way to test pagination functionality. Maybe first mount the component and expect the snapshot without doing any changes so as to track code changes and after that, you write another test to expect the state to change or onChangeLimit to be called as a way to assert what happens when you simulate change events? Otherwise good stuff @muhozi.

Reply:
Thanks, @dondrzzy for pointing this out. Let me add other tests for the rest events.